Change EC department accessory to fit on skirts

babydoll opened the pull request:
This removes the knee part of the EC department colour accessory so that it works on skirts. The department colour thing is probably part of the proper service uniform, so it should fit. It's currently impossible for it to fit when it has parts that attach on parts that don't exist on all of their uniforms.

Also: 
removes rolled accessory bits since no knees
renames gold service skirt for consistency

Resolves #18861

babydoll commented:
Guess I'll open a poll to see if people like the spaulders
https://baystation12.net/forums/threads/poll-ec-department-colour-things.5399/

Rebruiza commented:
Please embed an image of the pauldrons being worn on a skirt aswell. Thank you.

sabiram commented:
Those aren't in this PR.

ghost commented:
I realized. Regarding the pauldrons, they actually look good.

PurpleMartinJCK commented:
What didn't work for skirts?

babydoll commented:
Skirts do not have pant legs to attach a blue sticker to.

AtomicWorm commented:
It's odd. I always thought those leg markings were unsightly, but for some reason I don't really want to see them gone.

Gotta say I think the "pauldron" things don't look as good, though.

babydoll commented:
Poll wasn't very conclusive.

Also still ready for review

babydoll committed:
Change EC department accessory to fit on skirts

This removes the knee part of the EC department colour accessory
so that it works on skirts. The department colour thing is
probably part of the proper service uniform, so it should fit.
It's currently impossible for it to fit when it has parts that
attach on parts that don't exist on all of their uniforms.

babydoll commented:
Conflicts from chinsky's thing resolved

PsiOmegaDelta commented:
No complaints code-wise.

AtomicWorm commented on code/modules/clothing/under/accessories/solgov/department.dm:
```diff
@@ -3,7 +3,6 @@
 	name = "department insignia"
 	desc = "Insignia denoting assignment to a department. These appear blank."
 	icon_state = "dept_exped"
-	on_rolled = list("down" = "dept_exped_rolled")
```
This should actually be `on_rolled = list("down" = "none")`, or else when the outfit is rolled down, which removes the shirt, the department markings will remain visible.

babydoll replied:
Working on something else, will address shortly

babydoll commented:
Travis is playing nice now
